src/Aspire.Hosting.AWS/SdkUtilities.cs

```diff
@@ -45,7 +45,8 @@ internal static void ConfigureUserAgentString(object sender, RequestEventArgs e)
 
     internal static void ApplySDKConfig(EnvironmentCallbackContext context, IAWSSDKConfig awsSdkConfig, bool force)
     {
-        if (context.Logger != null)
+        var disableSdkValidation = (awsSdkConfig as AWSSDKConfig)?.DisableSdkValidation ?? false;
+        if (context.Logger != null && !disableSdkValidation)
```

Comment:
if we update the logic to be inverted (SDKValidationEnabled), then update this area too